chapters/solutions.Rmd

```diff
@@ -1187,15 +1250,24 @@ def set_plot_params(param_file):
                 mpl.rcParams[param] = value
 ```
 
-TODO: Add answers to the following questions:
-
-- Which file format do you find easier to work with?
-- What other factors should influence your choice of a configuration file syntax?
+1.  Most people seem to find Windows INI files easier to write and read,
+    since it's easier to see what's a heading and what's a value.
 
+2.  However, Windows INI files only provide one level of sectioning,
+    so complex configurations are harder to express.
```

Comment:
Presumably there are reasons why YAML is now more popular than INI? Maybe we should mention them in the solution, otherwise it seems a little odd that we didn't use INI.